lib/arena.rb

```diff
@@ -1,2 +1,41 @@
 class Arena
+
+  attr_accessor :name, :gladiators
```

Comment:
could instead to `attr_reader` to only allow gladiators via `add_gladiator` method